src/learn/ExperienceModule.tsx

```diff
@@ -159,7 +163,11 @@ class ExperienceModule extends Component<ExperienceModuleProps, ExperienceModule
     this.setState({ user: athleteProfile[0] });
   }
 
-  markSubmitted = async (milestone: any, githubLink: string, athleteNotes: string) => {
+  markSubmitted = async (
```

Comment:
Can you please type out the `milestone` object and replace the anys here?

Reply:
done